src/app/public/modules/toast/toaster.component.scss

```diff
@@ -11,6 +11,5 @@
   position: fixed;
   padding-bottom: $sky-margin-double;
   padding-right: $sky-margin-double;
-  padding-top: 50px;
```

Comment:
Why are we making this visual change for this?

Reply:
This padding was causing a 50px area of empty space that caused propagation issues. For example: 1) It kept the user from clicking on the omnibar when the toasts were tall enough to reach it. 2) It allowed the flyout to close if you clicked above the toasts. In this scenario, even though it seemed you were clicking on the flyout, the flyout would close unexpectedly. I double checked with @Blackbaud-SteveBrush that this style was an artifact from older code and was no longer necessary.

Reply:
Ok. Did it actually cause a visual bump though? I noticed that you removed all the screenshots.

Reply:
Chatted offline